content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Reset Jenkins administrator password
+
+As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it.
+To cover this possibility we have steps to follow to reset the admin password.
+
+== Reset administrator password
+
+* Login to your Jenkins controller and edit the Jenkins configuration file (config.xml) inside your Jenkins folder.
+* Look for the **useSecurity** option and change it from **true** to **false**.
+* Save your file and close it.
+* Restart Jenkins service to apply changes. (This time it will not ask for password)
+* On the dashboard, click on **Manage Jenkins** in the navigation pane on the left side of the page.
+* In Manage Jenkins, in the **Security** section you will get **Configure Global Security**. Click on it.
+* In **Security Realm**, click on the checkbox of **Jenkins' own user database** and save it. You will get redirected to the Manage Jenkins page.
+* In **Security**, click on **Manage Users**. 
+
+You will get a screen showing your user name with multiple tabs. Click on the setting icon.
+* Search for **Password** section to change your password.
+After changing the password you will be able to try logging into Jenkins again using the username and password that you have set.
```

Comment:
```suggestion
* Sign in to your Jenkins controller and edit the Jenkins configuration file (config.xml) inside your Jenkins folder.
* Look for the *useSecurity* option and change it from *true* to *false*.
* Save your file and close it.
* Restart the Jenkins service to apply your changes, and you can sign in without being asked for a password.
* On the dashboard, select *Manage Jenkins* in the navigation pane on the left side of the page.
* On the *Manage Jenkins* page, under *Security*, select *Configure Global Security*.
* In the *Security Realm* dropdown, select *Jenkins' own user database* and save.
* From the *Manage Jenkins* page, select *Manage Users* under *Security*.
* Select the gear icon to navigate to your user configuration page.
* Navigate to the *Password* section to change your password.
* Select *Apply* and then *Save*,  to save your changes and go back to the user *Status* page.

After changing the password, you can try logging into Jenkins again using the username and updated password.
```
Suggestion to make the instructions a bit more clear/direct.

Also, if these are the ordered steps to take, would it make more sense to user an [ordered list formatting](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/)?